src/frontend/components/Chat/SharedChatComponents/InputBar/index.tsx

```diff
@@ -50,13 +63,14 @@ export const InputBar = ({
       round="6px"
     >
       <TextInput
+        disabled={desactivateForm}
         focusIndicator={false}
         onChange={(event) => setInputText(event.target.value)}
         onKeyPress={handleKeypress}
         placeholder={
           <Box style={{ display: 'inline-grid' }} width="100%">
             <Text
-              color="blue-chat"
+              color={desactivateForm ? 'rgba(10, 103, 222, 0.3)' : 'blue-chat'}
```

Comment:
you should define this color in theme

Reply:
I don't want to define it in colors, because it's kind of hack to emulate a 'desactivate' color